lang/cs/Org.Apache.REEF.Client/YARN/YARNREEFClient.cs

```diff
@@ -28,19 +28,19 @@
 
 namespace Org.Apache.REEF.Client.YARN
 {
-    internal sealed class YARNClient : IREEFClient
+    internal sealed class YARNREEFClient : IREEFClient
```

Comment:
This renaming is to make the name less ambiguous after creation of YARN REST client. The class is internal so it is not a breaking change.